ci_scripts/build_external_tfm.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+"""
+Copyright (c) 2021 ARM Limited. All rights reserved.
+
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from os.path import join, isdir
+import argparse
+import sys
+import signal
+import logging
+import shutil
+
+sys.path.append("../")
+from psa_builder import *
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[Build-external-TF-M] %(asctime)s: %(message)s.",
+    datefmt="%H:%M:%S",
+)
+
+
+def _copy_tfm_binaries(args, suite):
+    """
+    Copy TF-M binaries from source to destination
+    :param args: Command-line arguments
+    :param suite: Test suite for PSA compliance
+    """
+
+    src_folder = join(
+        TF_M_BUILD_DIR, "trusted-firmware-m", "cmake_build", "bin"
+    )
+    dst_folder = join(
+        ROOT, "BUILD", args.mcu, TC_DICT[args.toolchain], "TFM", suite
+    )
+
+    logging.info("Copying folder: " + src_folder + " - to - " + dst_folder)
+    if not isdir(dst_folder):
+        os.makedirs(dst_folder)
+    for f in os.listdir(src_folder):
+        if os.path.isfile(join(src_folder, f)):
+            shutil.copy2(join(src_folder, f), join(dst_folder, f))
+
+    erase_flash_storage(
+        args.mcu,
+        suite,
+        "tfm_s_ns_signed.bin",
+        dst_folder
+    )
```

Comment:
This doesn't work, because you also need `bl2.bin`. I can think of two options:
* This script already copies the individual output binaries, which is enough for users to do manual concatenation (if less user friendly)
* Run the `srec_cat` command directly

Either way we can drop the commit to refactor `erase_flash_storage`.